### PR TITLE
User views

### PR DESCRIPTION
**Pull request by sbsmith86**
Use :hammer: Forge to style User CRUD views.

cc @katiecrane

### CODE REVIEW

File: resources/views/users/create.blade.php [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
 
-	<div>
-		{!! Form::label('slack_name', 'Slack Name: @') !!}
-		{!! Form::input('text', 'slack_name', null, ['placeholder' => 'whatever']) !!}
-	</div>
+@section('content')
 
-	<div>
-		{!! Form::label('start_date', 'Start Date: ') !!}
-		{!! Form::input('date', 'start_date', null, ['placeholder' => 'MM/DD/YYYY']) !!}
```
Comment (katiecrane) [inline resources/views/users/create.blade.php:15]: image

I'm looking into what changed, but it's expecting YYYY-MM-DD

Comment (sbsmith86) [inline resources/views/users/create.blade.php:15]: @katiecrane yea i changed this from a date field to a text field cause I was having issues pre-populating the date field with the users start date. I made an issue for it #21 which can be worked on once this gets merged in

Comment (sbsmith86) [inline resources/views/users/create.blade.php:15]: we either need to make it a date field again, or change the date format after the request is sent and before it gets stored

Comment (katiecrane) [inline resources/views/users/create.blade.php:15]: @sbsmith86 oh nice! I can work on that once it gets merged if you want

**Comment by sbsmith86** [comment]
ima just merge this to move :fast_forward:

**Pull request merged**